src/views/preview/preview.jsx

```diff
@@ -382,6 +384,12 @@ class Preview extends React.Component {
                         assetHost={this.props.assetHost}
                         backpackOptions={this.props.backpackOptions}
                         basePath="/"
+                        canCreateNew={this.props.canCreateNew}
+                        canRemix={this.props.canRemix}
+                        canReport={this.props.canReport}
```

Comment:
Whoops — just noticed this now after reviewing the GUI PR. We don't need to tell GUI about this.

Reply:
Right.